### PR TITLE
pregame summary 

### PR DESCRIPTION
**Pull request by AkifRafique**
added command /pregamesummary that displays ranks and champs of user currently playing a match

--I will resolved merge conflicts tomorrow.

### CODE REVIEW

File: client/league.py [hunk 1]
```diff
@@ -34,4 +34,23 @@ async def display_best_champs(message : Message) -> None:
...
+
+    @staticmethod
+    async def display_game_pregame_summary(message : Message) -> None:
+        summoner_name = re.compile("/pregamesummary (.*)").match(message.content).group(1)
+        summoner = Summoner(name=summoner_name, region="NA")
+        player_match_history = summoner.current_match()
+        
+        output = "```"
+        output += "{:>18} {:>14} {:>14} {:>15}".format("SUMMONER", 
+                                                                "KILLS",
```
Comment (svapnil) [inline client/league.py]: What does kills here do?

File: helper/league.py [hunk 2]
```diff
@@ -27,4 +27,36 @@ def display_team_info(team : LazyList) -> str:
...
+        name = player.summoner.name
+        champion = player.champion.name
+        soloRank = ""
+
+        flexRank = ""
+        try:
+            rank = [player.summoner.ranks[Queue.ranked_solo_fives].tier][0]
+            divison = [player.summoner.ranks[Queue.ranked_solo_fives].division][0]
+            soloRank = str(rank) + " " + str(divison)
+        except:
```
Comment (svapnil) [inline helper/league.py:70]: Is there a better way you can handle the not ranked case rather than expecting an exception? Just wondering

**Review by svapnil — COMMENTED** [review]
Looks almost perfect, just left some changes

File: client/league.py (hunk 1 above)
Comment (AkifRafique) [reply client/league.py]: I thought I changed it, It is supposed to show the champion being played by summoner.

File: helper/league.py (hunk 2 above)
Comment (AkifRafique) [reply helper/league.py:70]: So if the summoner does not have a rank, the api throws an exception. That is why I am using "try except". If you have better solution to handle this case I can change it to that.

**Comment by svapnil** [comment]
looks good

**Pull request merged**